xio-core/src/main/java/com/xjeffrose/xio/client/ClientConfig.java

```diff
@@ -74,4 +92,78 @@ public static ClientConfig fromConfig(String key, Config config) {
   public static ClientConfig fromConfig(String key) {
     return fromConfig(key, ConfigFactory.load());
   }
+
+  public static ClientConfig.Builder newBuilder(ClientConfig fallbackObject) {
+    return new ClientConfig.Builder(fallbackObject);
+  }
+
+  /**
+   * Used to create a ClientConfig at runtime.
+   *
+   * <p>If a value is not set, it defaults to using the default object's value.
+   */
+  public static class Builder {
+    private ClientConfig fallbackObject;
```

Comment:
The name is fallbackObject the javadoc says default object. I would pick one name and stick with it.

Reply:
fixed